cmd/net.go

```diff
@@ -77,18 +77,58 @@ func getHostIP4(host string) (ipList set.StringSet, err error) {
 	return ipList, err
 }
 
+// byLastOctetValue implements sort.Interface used in sorting a list
+// of ip address by their last octet value.
+type byLastOctetValue []net.IP
+
+func (n byLastOctetValue) Len() int      { return len(n) }
+func (n byLastOctetValue) Swap(i, j int) { n[i], n[j] = n[j], n[i] }
+func (n byLastOctetValue) Less(i, j int) bool {
+	return []byte(n[i].To4())[3] > []byte(n[j].To4())[3]
+}
```

Comment:
If the stated purpose is to not prefer the loopback address as the first entry, this patch (obviously) does not work for all cases - e.g.

```
127.0.0.1
10.0.0.1
```

Some test code for the above case here - https://play.golang.org/p/26yn5fAGMi

If we really shouldn't print 127.0.0.1 as the first IP, then an appropriate way to sort would be something like:

```go
func (n byLastOctetValue) Less(i, j int) bool {
	if n[i].String() == "127.0.0.1" {
		return false
	}
	if n[j].String() == "127.0.0.1" {
		return true
	}
	return []byte(n[i].To4())[3] > []byte(n[j].To4())[3]
}
```